bin/npmgraph.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+const path = require('path');
+const express = require('express');
+const open = require('open');
+const argv = require('minimist')(process.argv.slice(2));
+
+const app = express();
+const port = 3003;
+let packageName = '';
+
+const getPackageName = (packlist = []) => {
+  if (packlist.length > 0) return packlist[0];
+  console.log('Please assign packageName');
+};
+
+//handle cli params
+if (argv.local) {
+} else {
+  packageName = getPackageName(argv._);
+  if (!packageName) return;
```

Comment:
Bash scripts should exit with a non-zero code anytime there's an error.  Also, error and action related to this should happen together.

```suggestion
 if (!packageName) {
   console.log('Please assign packageName');  // or usage()? see below
   process.exit(1);
 }
```

... and actually even that can be improved. If we're adding support for a `--port` flag we should document that it's a thing, so have a usage function like so:
```
function usage() {
  console.log(`Usage: npmgraph [--port=portnum] package_name
  --port=portnum:  server port to listen on');
  packaage_name: package to start dependency graph at`);
}
```

Reply:
Thanks, I have learned a lot.